sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/Helpers/FlowAnalysis/ControlFlowGraphTest.cs

```diff
@@ -316,6 +316,52 @@ public void Cfg_NestedDoWhile()
             exitBlock.PredecessorBlocks.Should().OnlyContain(trueBranchBlock);
         }
 
+        [TestMethod]
+        [TestCategory("CFG")]
+        public void Cfg_DoWhile_Continue()
+        {
+            var cfg = Build(@"void fun(){

```

Comment:
I just copied example verbatim from the ticket.

Will rememeber that in future.